src/database/connection.js

```diff
@@ -0,0 +1,232 @@
+  },
+  gender: {
+    type: Sequelize.STRING,
+    allowNull: true
+  },
+  age: {
+    type: Sequelize.INTEGER,
+    allowNull: true
+  },
+  marital_status: {
+    type: Sequelize.STRING,
+    allowNull: true
+  },
+  numberOfCalls: {
+    type: Sequelize.INTEGER,
+    allowNull: true
+  },
+  involvment: {
+    type: Sequelize.STRING,
+    allowNull: true
+  }
+
+}, 
+  {
+  sequelize,
+  modelName: 'caller',
+  timestamps: false
+  // options
+});
+
+class Call extends Model {}
+Call.init({
+  // attributes
+  call_id: {
+    type: Sequelize.INTEGER,
+    //allowNull: false,
+    primaryKey: true, 
+    autoIncrement: true
+  },
+  typeOfContact: {
+    type: Sequelize.STRING,
+    allowNull: true
+  },
+  day: {
+    type: Sequelize.STRING,
+    allowNull: true
+  },
+  time: {
+    type: Sequelize.STRING,
+    allowNull: true
+  },
+  callDuration: {
+    type: Sequelize.STRING,
+    allowNull: true
+  },
+  typeOfCall: {
+    type: Sequelize.STRING,
+    allowNull: true
+  },
+  volonteer: {
+    type: Sequelize.INTEGER,
+    references: {
+
+    // This is a reference to another model
+    model: 'volonteer',
+   
+    // This is the column name of the referenced model
+    key: 'volonteer_id'
+    }
+  },
+  caller_id:{
+    type: Sequelize.INTEGER,
+    references: {
+
+    // This is a reference to another model
+    model: 'caller',
+   
+    // This is the column name of the referenced model
+    key: 'caller_id',
+  },
+  allowNull: true
+}
+
+}, 
+  {
+  sequelize,
+  modelName: 'call',
+  timestamps: false
+  // options
+});
+
+class CallDescription extends Model {}
+CallDescription.init({
+  // attributes
+  callDescription_id: {
+    type: Sequelize.INTEGER,
+    //allowNull: false,
+    primaryKey: true, 
+    autoIncrement: true
+  },
+  typeOfProblem: {
+    type: Sequelize.STRING,
+    allowNull: true
+  },
+  risk: {
+    type: Sequelize.STRING,
+    allowNull: true
+  },
+  previousAttempts: {
+    type: Sequelize.STRING,
+    allowNull: true
+  },
+  content: {
+    type: Sequelize.STRING,
+    allowNull: true
+  },
+  note: {
+    type: Sequelize.STRING,
+    allowNull: true
+  },
+  call_id:{
+    type: Sequelize.INTEGER,
+    references: {
+
+    // This is a reference to another model
+    model: 'call',
+   
+    // This is the column name of the referenced model
+    key: 'call_id',
+  },
+  allowNull: true
+}
+
+}, 
+  {
+  sequelize,
+  modelName: 'callDescription',
+  timestamps: false
+  // options
+});
+
+class Volonteer extends Model{}
+Volonteer.init({
+  volonteer_id: {
+    type: Sequelize.INTEGER,
+    primaryKey: true,
+    autoIncrement: true
+  },
+  name: {
+    type: Sequelize.STRING,
+    allowNull: true
+
+  },
+  surname :{
+    type: Sequelize.STRING,
+    allowNull: true
+
+  },
+  birth:{
+    type: Sequelize.DATE,
+    allowNull: true
+
+  },
+  city:{
+    type: Sequelize.STRING,
+    allowNull: true
+
+  }},
+  {
+    sequelize,
+    modelName: 'volonteer',
+    timestamps: false
+});
```

Comment:
This is not what I have asked for when I mentioned a new file.
Every entity needs to be a separate file.